src/test/java/io/debezium/connector/informix/InformixConnectorIT.java

```diff
@@ -532,7 +532,7 @@ public void verifyOffsets(boolean withOnlineUpd) throws Exception {
 
         // Wait for streaming to start
         waitForStreamingRunning(TestHelper.TEST_CONNECTOR, TestHelper.TEST_DATABASE);
-        waitForAvailableRecords(1, TimeUnit.MINUTES);
+        waitForAvailableRecords(2, TimeUnit.MINUTES);
```

Comment:
So this doesn't have to be done as a part of this specific PR, but I'd prefer we have a method that doesn't take any time arguments and we instead source the time values from a system property that has a reasonable default.  This way it becomes really simple for us to tweak these wait times on GitHub actions if we find there are time-related issues without needing to directly touch the source code for the tests.  

wdyt @nrkljo ?

Reply:
Yes, that sounds reasonable. Something akin to `debezium.test.records.waittime`?

Reply:
@nrkljo I don't think a new option needs to be introduced. Frequently it is enough to re-use `debezium.test.records.waittime` and multiply it with some factor.